src/__tests__/pages/api/auth/signup.test.ts

```diff
@@ -0,0 +1,238 @@
+/**
+ * @jest-environment node
+ */
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import { PrismaClient } from '@prisma/client';
+import signUpHandler from '../../../../pages/api/auth/signup';
+import {
+  mockUserData,
+  mockManyUserData
+} from '../../../../__mocks__/pages/api/auth';
+import type { UserDataProp } from '../../../../__mocks__/pages/api/auth';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const prisma = new PrismaClient();
+
+beforeAll(async () => {
+  await prisma.$connect();
+});
+
+beforeEach(async () => {
+  await prisma.user.deleteMany();
+});
+
+afterAll(async () => {
+  await prisma.$disconnect();
+});
+
+describe('Successfully create user(s)', () => {
+  it('Create user successs', async () => {
```

Comment:
these strings really should read like sentences - "it create user success" is a weird sentence. For example, "it successfully creates a user" is a better sentence, which becomes `it('successfully creates a user', ...)`
This comment applies to every PR/every test